apps/frontend/src/pages/Search/subcomponents/InstructionUTR/Instruction.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import s from "./Instruction.module.scss";
+
+const Instruction: React.FunctionComponent = () => {
+	return (
+		<div className={s.maindiv_instruction}>
+			<p className={s.instruction}>
+				Wähle aus welche Dienstleistung du benötigst und gib deinen Standort an
+			</p>
+		</div>
+	);
+};
+
+export default Instruction;
```

Comment:
wenn der inhalt so wenig ist, macht es manchmal keinen Sinn, extra eine Komponente anzulegen. Das könnte man theoretisch auch einfach in die Suchen-Seite einfügen. Muss aber von mir aus für den PR nicht mehr geändert werden

Reply:
Ja, stimmt hast du recht. Hintergrund war am Anfang, dass ich dachte ich würde die Komponente auf mehreren Seiten wiederverwenden, bevor ich gemerkt habe das das alles eine Seite ist die sich nur dynamisch ändert.